RecoVertex/Configuration/python/RecoVertex_phase2_timing_cff.py

```diff
@@ -57,3 +57,6 @@
 from SimTracker.TrackAssociatorProducers.quickTrackAssociatorByHits_cfi import quickTrackAssociatorByHits
 from SimTracker.TrackAssociation.trackTimeValueMapProducer_cfi import trackTimeValueMapProducer
 from CommonTools.RecoAlgos.tofPID_cfi import tofPID
+
+tofPIDfor4DwithPID=tofPID.clone()
+tofPID.vtxsSrc=cms.InputTag('unsortedOfflinePrimaryVertices4DwithPID')
```

Comment:
please clarify the logic used for naming these modules.
Apparently ```tofPIDfor4DwithPID``` is used to compute tof PID using vertices without PID, which is later used downstream to make 4D vertices with PID and then compute a better set of tof PID in ```tofPID```.
It seems more clear to me to call the first  ```tofPIDwith4DnoPID```.

For the last line, please avoid modifications outside of the initial cff. If unsortedOfflinePrimaryVertices4DwithPID is the new default, just change it in the TOFPIDProducer::fillDescriptions.